payment-processor/src/integration-tests/kotlin/org/ostelco/prime/paymentprocessor/StripePaymentProcessorTest.kt

```diff
@@ -74,7 +76,7 @@ class StripePaymentProcessorTest {
     @After
     fun cleanUp() {
         val resultDelete = paymentProcessor.deletePaymentProfile(stripeCustomerId)
-        assertEquals(true, resultDelete.isRight())
+        assertNotFailure(resultDelete)
```

Comment:
You can also do something like this I guess?
```
check(resultDelete == ProfileInfo(customerId).right())
```
That will also verify that the data is as expected.

We also have [https://github.com/kotlintest/kotlintest](https://github.com/kotlintest/kotlintest), which could be something to look into?